### ISSUE TITLE
Stem glyphs missing?

### ISSUE THREAD

**Issue by Jamesqiu-123** [MuseScoreFonts/Leland#27]
I have no experience using music fonts but I notice that the character for stem(U+E210) is seemed missing from the leland font, is it true?

**Comment by oktophonie** [comment]
Indeed. It's not needed generally in music notation software as the stems are drawn as primitive lines, rather than using the stem glyph which has fixed dimensions.
